packages/blockchain-link/src/index.ts

```diff
@@ -70,6 +70,9 @@ class BlockchainLink extends TypedEmitter<Events> {
 
     private throttler: Throttler;
 
+    // worker promise is used to prevent multiple workers initialization when multiple methods are called at called parallel before worker is initialized
```

Comment:
```suggestion
    // worker promise is used to prevent multiple workers initialization when multiple methods are called at parallel before worker is initialized
```

Just a typo. Otherwise looks good 🚀